src/pages/serverError/page.tsx

```diff
@@ -0,0 +1,17 @@
+import style from "./serverError.module.css";
+
+export default function ServerErrorPage() {
+	return (
+		<div className={style.server_error}>
+			<div className={style.server_error__text_wrapper}>
+				<h1 className={style.server_error__title}>Ошибка на сервере...</h1>
+				<p className={style.server_error__text}>Приложение еще активно разрабатывается и ошибки случаются...</p>
+				<p className={style.server_error__text}>
+					Пока Вы это читаете, по тревоге были подняты наши программисты и администраторы, которые уже прочитали
+					пожробный отчет об этой ошибке и приступили к ее устранению.
+				</p>
```

Comment:
to avoid syntax errors, you can copy the text directly from figma and paste it into jsx